compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -750,6 +750,7 @@ class Definitions {
   @tu lazy val RefiningAnnotationClass: ClassSymbol = ctx.requiredClass("scala.annotation.RefiningAnnotation")
 
   // Annotation classes
+  @tu lazy val ScalaAnnotationInternal: ClassSymbol = ctx.requiredClass("scala.annotation.internal")
```

Comment:
Why do we need this? It doesn't seem like we are using it in the codebase, can we drop it?

Reply:
yes that was added for something I later removed, thanks